### PR TITLE
Fix issue template

### PR DESCRIPTION
**Pull request by ybw0014**
The bug report issue template does not work right now.

### CODE REVIEW

**Comment by ybw0014** [comment]
I will add a suggestion template as well.

**Review by Sefiraat — DISMISSED** [review]

**Comment by Sefiraat** [comment]
I can’t work out why it’s having trouble building when the only thing that’s changed here is this one file and yet the last merge a week ago is a-ok. I’ll take a more in depth look later

**Review by Sefiraat — CHANGES_REQUESTED** [review]
Needs to build. I’ll look into this

File: src/main/java/io/github/sefiraat/slimetinker/items/BaseItem.java [hunk 1]
```diff
@@ -20,7 +21,8 @@ public BaseItem(String id, String texture, String name, String... lore) {
     }
 
     @Override
-    public @NotNull String getId() {
+    public @Nonnull
+    String getId() {
```
Comment (ybw0014) [inline src/main/java/io/github/sefiraat/slimetinker/items/BaseItem.java:25]: here

**Review by Sefiraat — APPROVED** [review]

**Pull request merged**